### PR TITLE
CIP 2: Professional Verifier Pool

### PR DESCRIPTION
**Pull request by nambrot**
In the early prototypes of the [lightweight identity protocol](https://docs.celo.org/celo-codebase/protocol/identity), Celo had a concept of a verifier app. It would send text messages on behalf of the protocol and would receive a share of the attestation fee revenue. It was a truly permissionless on-ramp. However, over discussions, we had sybil and game-theoretic concerns over the viability of that scheme. This CIP proposes a return of the verifier app in the context of a professionally run text message provider that uses verifier apps to send the message. In this scheme, the provider has a reputation to uphold and acts as a possible alternative to providers like Twilio.

### CODE REVIEW

**Comment by asaj** [comment]
@nambrot can we close?

**Comment by nambrot** [comment]
I'd say this CIP is still open so I'd actually argue we should merge it as a draft?

File: CIPs/0002.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# CIP 2: Professional Verifier Pool
```
Comment (YazzyYaz) [inline CIPs/0002.md:1]: Please ensure your title follows the new cip-template.md file.

Also ensure status is Draft.

https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also please ensure the rest of the document headings are following the new CIP template document. Abstract, Simple Summary, Motivation, Rationale, Specification, CopyRight, Implementation, etc.

File: CIPs/0002.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- `SENT` - The verifier has claimed to have sent the message
+- `CONFIRMED` - The user has completed the attestation
+- `REWARDED` - The verifier has been rewarded
+
+
+**Build a text message provider + app**
+We should effectively revive the pool part of the old verification pool and the verifier app, and adjust it to the new interface to work as a module to the attestation service. Since this service is not expected to run by validators, but by specific entities, building this on top of proprietary tooling is likely ok. We could provide sybil resistance through dedicated APIs such as [https://developer.android.com/training/safetynet/attestation](https://developer.android.com/training/safetynet/attestation) or [https://developer.android.com/training/articles/security-key-attestation](https://developer.android.com/training/articles/security-key-attestation). If we deem these APIs to not be broadly available in our target markets, we could fallback to classic sybil mechnisms like Social Networks, although they are 1. not as sybil resistent and 2. not as permissionless/inclusive.
+
+
+### Alternative Solution
```
Comment (YazzyYaz) [inline CIPs/0002.md:90]: I'd like to see this whole document following the new template so we can have consistency.

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

**Review by prestwich — CHANGES_REQUESTED** [review]
To be merged please:
1. Update to follow the new template
2. Open a discussions issue `CIP 2 - Professional Verifier Pool - Discussions`
3. Update the discussions-to link in the new template header

**Comment by nambrot** [comment]
I'll close as not relevant

**Pull request closed without merging**